src/l1-resources/stream-connection/README.md

```diff
@@ -0,0 +1,69 @@
+# stream-connection
+
+The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
+
+> AWS CDK [L1 construct] and data structures for the [AWS CloudFormation Registry] type `MongoDB::Atlas::StreamConnection`.
```

Comment:
Doesn't look like this is correctly formatted

Reply:
Fixed